### ISSUE TITLE
Accessibility - Content missed when using screen reader technology

### ISSUE THREAD

**Issue by SteveLindsay461** [MHRA/products#202]
Raised by Kyle
**Describe the bug**
Using the Chrome plugin ‘ChromeVox Classic Extension’, the following content is ‘skipped’ over i.e. using this screen reader technology, it is never read out.

Homepage
•	‘Or by find by active substance’ (instead the A-Z list is read out, but without any context that it’s only for active substances)
•	Yellow Card: this goes straight to ‘Make a report’: it does not read out any links from the images next to this, so misses the context as to what users are making a report to
•	All descriptive text on the homepage – after the Yellow Card section it skips all the remaining text until it gets to the ‘Safety Public Assessment Reports’ page. 
Disclaimer
•	All descriptive text is missed out, other than those (like ‘European Medicines Agency’) which are links
•	The Disclaimer is not read out at all
Results page
•	The descriptive text at the beginning is not read out
•	The ‘PAR’/’SPC’/’PIL’ (the blue box next to the listing) is not read out at all
•	If conducting a search, the user has to tab back through the entire A-Z list before reaching the results

**Comment by craiga** [comment]
[ChromeVox Classic Extension is available here](https://chrome.google.com/webstore/detail/chromevox-classic-extensi/kgejglhpjiefppelpmljglcjbhoiplfn?hl=en).

Note the following from [the extension's overview](https://chrome.google.com/webstore/detail/chromevox-classic-extensi/kgejglhpjiefppelpmljglcjbhoiplfn?hl=en):

> Please note that this extension is now in maintenance mode; no new additions will be made moving forward.

**Comment by craiga** [comment]
@SteveLindsay461 All of the text mentioned in this ticket is being read out when using ChromeVox Classic Extension, but I may be misusing it. If this issue is still occurring, can you show me how to replicate it?
